pkg/typesystem/typesystem.go

```diff
@@ -556,18 +556,17 @@ func isUsersetRewriteValid(
 }
 
 func validateRelationTypeRestrictions(model *openfgapb.AuthorizationModel) error {
-	t := New(model)
-	allTupleToUsersetDefinitions := t.GetAllTupleToUsersetsDefinitions()
+	typeSystem := New(model)
```

Comment:
Minor nit pick - we've been using `typesys` elsewhere. Mind either changing it here to `typesys` or `typeSystem` elsewhere?

Reply:
Done. Thanks!